cmd/root/bundle.go

```diff
@@ -109,8 +112,48 @@ func configureBundle(cmd *cobra.Command, b *bundle.Bundle) (*bundle.Bundle, diag
 	return b, diags
 }
 
-// MustConfigureBundle configures a bundle on the command context.
-func MustConfigureBundle(cmd *cobra.Command) (*bundle.Bundle, diag.Diagnostics) {
+func chdirRewrite(root string, paths ...*string) error {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	if cwd == root {
+		return nil
+	}
+
+	if testing.Testing() {
+		// integration tests run cmd code, Chdir() messes up parallel tests.
+		return nil
+	}
+
+	if err := os.Chdir(root); err != nil {
+		return err
+	}
+
+	for _, pathPtr := range paths {
+		if pathPtr == nil || *pathPtr == "" {
+			continue
+		}
+
+		if filepath.IsAbs(*pathPtr) {
+			continue
+		}
+
+		abs := filepath.Join(cwd, *pathPtr)
+		rel, err := filepath.Rel(root, abs)
```

Comment:
This will fail if we provide `DATABRICKS_BUNDLE_ROOT` which is outside of CWD.  Can we have a test for this to confirm setting DATABRICKS_BUNDLE_ROOT works as expected?